contrib/buildsystems/engine.pl

```diff
@@ -40,6 +41,7 @@ sub showUsage
 # Parse command-line options
 while (@ARGV) {
     my $arg = shift @ARGV;
+	#print "Arg: $arg \n";
```

Comment:
If you want to keep it as deactivated code, you should make it truly deactivated code: guard it by some environment variable or some such. I.e. something like this:

```perl
print "Arg: $arg\n" if ($ENV{'DEBUG_MSVC_BUILD'});
```